apps/super-app/src/common/Theme.ts

```diff
@@ -59,6 +64,9 @@ const callHomeTheme = createTheme(
       grey: colors.grey,
       error: colors.error,
       warning: colors.yellow,
+      text: {
+        primary: colors.text.primary,
```

Comment:
I don't get this, I tried and it doesn't work. Also added `main` field into the `colors` object but it doesn't work either.

Reply:
What did you try specifically? I changed this to

```
      text: colors.text,
```

and it compiles just fine for me. I'd expect it to work for you too, because `colors.text`(what I used) and `{ primary: colors.text.primary }` (what you used in this PR) have exactly the same type.

Reply:
![image](https://user-images.githubusercontent.com/29354437/228863991-b8ccd1f9-c955-4e30-b079-451e42db24f3.png)

Reply:
I see, so you had:

```
text: {
  primary: colors.text
}
```

but what I meant was:

```
text: colors.text
```